cardiac.py

```diff
@@ -1,11 +1,11 @@
 
+from random import *
+import sys
+import pygame
+import pickle
+from gi.repository import Gtk
 import gi
 gi.require_version('Gtk', '3.0')
```

Comment:
The import of `gi` and use of `gi.require_version` should come before any other imports from `gi`.